src/types/types.ts

```diff
@@ -70,4 +70,21 @@ export type AttendeeBasicInformation = BasicInformation & {
     diet: string;
     heardFrom: string;
     faculty: string;
-  };
\ No newline at end of file
+  };
+
+export type AttendeeDetailedInformation = {
```

Comment:
Types should be consistent with db. So i think this here should be "Registration"